example/sync.config.js

```diff
@@ -1,6 +1,6 @@
 export const SYNC_CONFIG = {
   // Set `enabled` to `true` to enable sync.
-  enabled: false,
+  enabled: Boolean(process.env.TEMPLATE_SYNC_ENABLED) || false,
```

Comment:
Maybe we can be explicit about what value we expect? I don't mind too much though!
```suggestion
  enabled: process.env.TEMPLATE_SYNC_ENABLED === '1' || false,
```